product_app/models.py

```diff
@@ -119,33 +119,32 @@ def __self__(self):
     class Meta:
         db_table = 'images'
 
-class ListThumnail(models.Model):
+class ListThumbnail(models.Model):
     image_url     = models.CharField(max_length = 1000)
     product_color = models.ForeignKey('ProductColor', on_delete = models.SET_NULL, null = True)
     
     class Meta:
-        db_table = 'list_thumnails'
+        db_table = 'list_thumbnails'
 
-class ProductcolorProductthumnail(models.Model):
+class ProductColorProductThumbnail(models.Model):
```

Comment:
ProductColorThumbnail 이라고만해도 되지않을까요 ? 
```suggestion
class ProductColorThumbnail(models.Model):
```